### PR TITLE
WIP: find common base for settings in all skin

### PR DESCRIPTION
**Pull request by nopeppermint**
some work towards
https://bugs.launchpad.net/mixxx/+bug/1740513

### CODE REVIEW

File: res/skins/Deere/mixer.xml [hunk 1]
```diff
@@ -235,7 +235,7 @@
               </WidgetGroup>
             </Children>
             <Connection>
-              <ConfigKey>[Deere],show_crossfader</ConfigKey>
+              <ConfigKey>[Master],show_xfader</ConfigKey>
```
[the remark this answers is not in the document]
Comment (daschuer) [reply res/skins/Deere/mixer.xml]: Ups, I see "[Master],show_xfader" already exists.

**Comment by daschuer** [comment]
We already have persistent controls with [Master] group. Changing this breaks skin settings fro some users. 
We break also skin settings when moving to the common Control Objects. 
Do we want to ignore this or do we need a solution?

**Comment by Be-ing** [comment]
I think if we have to break skin settings anyway, it kinda makes sense to move all skin settings CO names to good names at the same time. On the other hand, we don't have to break all skin settings. Breaking skin settings will be mildly annoying for users upgrading to 2.2, but it's easy to get them back to the user's preferred state.

**Comment by ronso0** [comment]
In [Master] group we have COs that store hardware/sound settings (gain, headphone), but also some COs that store skin options (show_4decks). IMO it would be ideal to have all skin options in [Skin]. This means we'd have to move some COs from [Master] as well as from [EffectRack1], [PreviewDeck], [Samplers] etc. to [Skin].
I'm wondering if it's worth the effort to do that (check source code and all mappings that use them) or just cumulate all 'new' skin COs in [Skin] like show_4effectunits and pretty much all new settings in Tango.

**Comment by daschuer** [comment]
All legacy controlls are there for a reason.
Moving all of them to [Skin] group is overdone for me. So I would prefere a balanced way, break some skins, to unity the names, but not move all skin related COs

**Comment by ronso0** [comment]
There are some conflicts now after merging the recent skin tweaks.
@nopeppermint Do you want to solve them and continue?
Otherwise I'd a PR from a branch I started a while ago where I unified COs like this:
`[Skin],show_spinnies`
`[Skin],show_coverart`
`[Skin],show_8_hotcues`
`[Skin],show_big_spinny_cover`
`[Deere/Tango],sampler_row_N` > `[Skin],show_N_sampler_rows` (somehow make this 1-indexed?)
`[SamplerRowN],expanded` > `[Skin],sampler_row_n_expanded`
`[Deere],show_starrating`/`[Tango],stars` > `[Skin,show_starrating]`

It would be nice to have consistent skin COs in 2.2

**Comment by nopeppermint** [comment]
@ronso0 
you can take this

**Pull request closed without merging**